.github/workflows/CI.yml

```diff
@@ -32,15 +32,15 @@ jobs:
         uses: actions/checkout@v2
         with:
           repository: 'opensearch-project/OpenSearch'
-          ref: '1.x'
+          ref: '1.1'
           path: OpenSearch
       - name: Build OpenSearch
         working-directory: ./OpenSearch
-        run: ./gradlew publishToMavenLocal
+        run: ./gradlew publishToMavenLocal -Dbuild.snapshot=false
```

Comment:
Are we going to have to toggle this everytime we cut a branch?

Reply:
From next version onwards, we will be using sonatype. Hence, this might be last time we have to checkout OpenSearch and publish to local Maven.

Reply:
I think your CI should continue building a snapshot. This is to disambiguate with the bundle build of the entire distribution.

Reply:
Ack